Week18/App/server.js

```diff
@@ -0,0 +1,87 @@
+const express = require('express');
+const mysql = require('mysql2');
+
+const port = 3000;
+const app = express();
+
+const pool = mysql.createConnection({
+  host: 'localhost',
+  user: 'root',
+  password: 'FGSz',
+  database: 'my_app',
+  debug: false,
+});
+
+pool.connect((err) => {
+    if (err) {
+        console.error(err);
+    } else {
+        console.log("Connection established");
+    }
+});
+
+// Serve files from the 'public' folder
+app.use(express.static('public'));
+
+// Convert JSON bodies to JavaScript objects
+app.use(express.json());
+
+app.get('/', (req, res) => {
+    // Load index.html from the public folder
+    res.redirect('/index.html');
+});
+
+app.post('/api/my_app', (req, res) => {
```

Comment:
Ugyanaz a probléma mint a tábla elnevezésnél; érdemes a resource-oknak nevet adni akkor is, ha egyelőre ez az egy van, ha esetleg később bővítenéd az alkalmazást, jól fog jönni.